template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/README.md.tmpl

```diff
@@ -1,5 +1,144 @@
 # {{template `project_name` .}}
 
+{{- if (eq .input_setup_cicd `no`) }}
+This directory contains an ML project based on the default
+[Databricks MLOps Stacks](https://github.com/databricks/mlops-stacks),
+defining a production-grade ML pipeline for automated retraining and batch inference of an ML model on tabular data.
+
+
+## Code structure
+This project contains the following components:
```

Comment:
This readme file is only relevant when users have done the step1 of "Setting up CI/CD for multiple projects" and before doing step2.
And its content is not consistent with project-overview.md. 
Let's avoid mentioning the intermediate state dir structure.

Reply:
The idea is that they won't have the `docs` folder for this project (since it's a top-level directory) which is fine since most of them deal with CI/CD. I copied the non-CI/CD relevant stuff to this project so I think the dir structure helps since it is project specific in this case.

And that is correct this file is relevant only after Step 1, but step 1 is generating the project so by definition they won't have this file until they do step 1. And step 2 is linking the CI/CD workflows, which is also the last step. If they do it without the help of this README awesome! And if not, well then that's what the README is for!

> The approach of copying the CICD files of the first project will not work well because it assumes that all the projects have the same CICD platform and feature store selections, which may not be the case.
For this I think we don't have to be too concerned about different CI/CD platform across projects since that's usually the point of a monorepo (and in general most companies only keep a single CI/CD platform across the whole company), but to address that and the feature store aspect, we can just ask them to regenerate the project and copy over only the workflow files (what we currently recommend).

Reply:
Since there's already the 2-step for generating a different project in a monorepo, it's less likely for users to pause and check the additional readme before doing step2. This doc block will almost always be obsolete immediately.

Reply:
Yeah this is more for the data scientists who just initialized the project without CI/CD (not necessarily following the 2-step from the other project)

Reply:
Updated the README with the CICD/FS note!